src/include/bootloader/grub/helps.rb

```diff
@@ -80,8 +80,12 @@ def initialize_bootloader_grub_helps(_include_target)
         ),
         "password"          => _(
           "<p><b>Protect Boot Loader with Password</b><br>\n" \
-            "Define the password that will be required to access the boot menu. YaST will only accept the password if you repeat\n" \
-            "it in <b>Retype Password</b>.</p>"
+            "Define the password that will be required to access the boot menu. \n" \
+            "YaST will only accept the password if you repeat\n" \
+            "it in <b>Retype Password</b>. If <b>Allow to boot locked default \n" \
+            "entry without password</b> is checked, then without password is \n " \
+            "possible to let bootloader autoboot default entry with its kernel \n" \
+            "parameters.</p>"
```

Comment:
> If Allow to Boot Default Entry is checked then that action will not require the password.
May be it helps to understand what GRUB2 actually implements.

You can enable authentication in GRUB2 (by setting superusers variable to list of user names). As soon as superusers variable is defined, entering CLI, editing menu entry and executing menu entry (i.e. usually booting) requires authentication as one of users in this list.

It is possible to either provide additional list of users that are allowed to execute individual menu entry (but not to edit it) or mark individual menu entry as "unrestricted" so executing it does not require authentication.

What currently is *NOT* possible is to autoboot default entry while keeping password protection for it. Either entry must be marked as unrestricted (and so always allowed to boot, manually or auto) or you need to enter password both for manual or autoboot.

Looking at it, autoboot without password looks like sensible feature request.